### ISSUE TITLE
Selecting imported MusicXML trill leads to crash (when Properties is opened)

### ISSUE THREAD

**Issue by rettinghaus** [musescore/MuseScore#25871]
Edited by @DmitryArefiev 

### Issue type

Crash or freeze

### Description with steps to reproduce

1. **Properties** panel should be opened
2. import a MusicXML with a `wavy-line` trill ([testTrills_ref.xml.zip](https://github.com/user-attachments/files/18171271/testTrills_ref.xml.zip))
3. select trill
4. MuseScore crashes

### Supporting files, videos and screenshots

E.g. a file from the tests:
[testTrills_ref.xml.zip](https://github.com/user-attachments/files/18171271/testTrills_ref.xml.zip)

See https://github.com/musescore/MuseScore/issues/25871#issuecomment-2561147469 

### What is the latest version of MuseScore Studio where this issue is present?

4.4.4, 4.5 nightly

### Regression

Yes, this used to work in **MuseScore 4.0.2**

### Operating system

macOS 15, Windows10

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by rettinghaus** [comment]
Maybe fixed with #25811

**Comment by Jojo-Schmitz** [comment]
So might be a duplicate of #23977

**Comment by oktophonie** [comment]
I don't get the crash in master (at least as of 19 Dec nightly) even without #25811 .

**Comment by miiizen** [comment]
Neither do I

**Comment by rettinghaus** [comment]
Tested again with latest nightly, and it still crashes. I tried the test file above and this one:  
[technical.xml.zip](https://github.com/user-attachments/files/18212330/technical.xml.zip)

**Comment by DmitryArefiev** [comment]
OK, I found the reason. The **Properties** panel should be opened when selecting trill

https://github.com/user-attachments/assets/6f240cdb-eea3-488c-80ca-5e70ce5d80a7


Occurs in **4.1.0 - 4.5**

**Comment by DmitryArefiev** [comment]
And it **doesn't crash** after you save the file in **mscz** format

**Comment by pacebes** [comment]
> And it **doesn't crash** after you save XML file in **mscz** format

I agree. It is fixed with this PR: https://github.com/musescore/MuseScore/pull/25811
